watertap/unit_models/reverse_osmosis_0D.py

```diff
@@ -205,46 +86,112 @@ def build(self):
+                )
 
         # Add additional state blocks
         tmp_dict = dict(**self.config.property_package_args)
         tmp_dict["has_phase_equilibrium"] = False
         tmp_dict["parameters"] = self.config.property_package
         tmp_dict["defined_state"] = False  # these blocks are not inlets
-        # Interface properties
-        self.feed_side.properties_interface_in = self.config.property_package.state_block_class(
+        # Build permeate side
+        self.permeate_side = self.config.property_package.state_block_class(
             self.flowsheet().config.time,
-            doc="Material properties of feed-side interface at inlet",
+            self.length_domain,
+            doc="Material properties of permeate along permeate channel",
             default=tmp_dict)
-        self.feed_side.properties_interface_out = self.config.property_package.state_block_class(
+        self.mixed_permeate = self.config.property_package.state_block_class(
             self.flowsheet().config.time,
-            doc="Material properties of feed-side interface at outlet",
+            doc="Material properties of mixed permeate exiting the module",
             default=tmp_dict)
-        self.permeate_side.properties_mixed = self.config.property_package.state_block_class(
+        # Interface properties
+        self.feed_side.properties_interface = self.config.property_package.state_block_class(
             self.flowsheet().config.time,
-            doc="Material properties of mixed permeate",
+            self.length_domain,
+            doc="Material properties of feed-side membrane interface",
             default=tmp_dict)
 
         # Add Ports
         self.add_inlet_port(name='inlet', block=self.feed_side)
         self.add_outlet_port(name='retentate', block=self.feed_side)
-        self.add_port(name='permeate', block=self.permeate_side.properties_mixed)
+        self.add_port(name='permeate', block=self.mixed_permeate)
 
         # References for control volume
         # pressure change
-        if (self.config.has_pressure_change is True and
+        if (self.config.has_pressure_change and
                 self.config.momentum_balance_type != 'none'):
             self.deltaP = Reference(self.feed_side.deltaP)
 
+        self._make_performance()
+
+        self._add_expressions()
+
+
+    def _make_performance(self):
+
+        units_meta = self.config.property_package.get_metadata().get_derived_units
+
+        solvent_set = self.config.property_package.solvent_set
+        solute_set = self.config.property_package.solute_set
+
+        self.rejection_phase_comp = Var(
+            self.flowsheet().config.time,
+            self.config.property_package.phase_list,
+            solute_set,
+            initialize=0.9,
+            bounds=(1e-2, 1 - 1e-6),
+            units=pyunits.dimensionless,
+            doc='Observed solute rejection')
```

Comment:
Why can't this be moved to the base class? Did the 1DRO model not have rejection?